Sentinel/Classes/Core/UIImage+Resize.swift

```diff
@@ -7,10 +7,20 @@
 
 import Foundation
 extension UIImage {
-    func resize(for size: CGSize = CGSize(width: 24, height: 24)) -> UIImage? {
+
+    static func loadImageFromBundle(name: String) -> UIImage? {
+        let frameworkBundle = Bundle(for: Sentinel.self)
+        guard let bundleURL = frameworkBundle.resourceURL?.appendingPathComponent("Sentinel.bundle")
+        else { return nil }
+        let resourceBundle = Bundle(url: bundleURL)
+        guard let image = UIImage(named: name, in: resourceBundle, compatibleWith: nil) else {return nil}
+        return image
+    }
```

Comment:
Do a bit of formatting here please :)

Reply:
@fgulan you are better than me with multi bundles, is all of this really necessary, it's seems really complicated?

Reply:
I think I wrote it a little bit better. In my opinion, it's a little easier to read than the previous solution